README.md

```diff
@@ -21,12 +21,16 @@ use my_crate::{function_under_test, is_valid, meets_specification};
 
 #[kani::proof]
 fn check_my_property() {
+
```

Comment:
I agree lines between these help, but I would remove this first one (most of the time, programs have a comment or declaration here).